tests/bootstrap.php

```diff
@@ -1,9 +1,5 @@
 <?php
 
-if (!$loader = require_once __DIR__.'/../vendor/autoload.php') {
-    die('You must set up the project dependencies, run the following commands:'.PHP_EOL.
-        'curl -s http://getcomposer.org/installer | php'.PHP_EOL.
-        'php composer.phar install'.PHP_EOL);
-}
```

Comment:
Now, it's done by phpunit [itself](https://github.com/sebastianbergmann/phpunit/blob/master/phpunit#L25-L32).